tests/integration/sync_test.rs

```diff
@@ -14,8 +14,8 @@ fn test_sync_down() {
     let includes: Option<Vec<String>> = Some(vec!["*.txt".to_string()]);
     let excludes: Option<Vec<String>> = Some(vec!["*".to_string()]);
 
-    let source = SyncParam::new_bucket(crate::TEST_BUCKET.to_owned(), s3_key.to_owned());
-    let destination = SyncParam::new_local(local_directory.to_owned());
+    let source = SyncParam::new_bucket(&crate::TEST_BUCKET.to_owned(), &s3_key.to_owned());
```

Comment:
Sometimes you can't just take the linter output verbatim, since we're trying to get rid of the `.to_owned()` calls here, you can just simply pass `crate::TEST_BUCKET` since it's already a `&str`, same with `s3_key` here.  If the type is a `String`, then pass `&var_name` and remove the `.to_owned()`.